spec/fabricators/associated_group_fabricator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
Please rename this file so it ends in `_spec.rb`

Reply:
Note that this one is already ``_fabricator``.